aqua/Aerial/Portfile

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               xcode 1.0
+PortGroup               github 1.0
+
+github.setup            JohnCoates Aerial 2.0.5 v
+
+categories              aqua
+license                 MIT
+platforms               darwin
+maintainers             {danchr @danchr} openmaintainer
+
+description             Apple TV Aerial Screensaver for Mac
+
+long_description        Aerial is a Mac screensaver based on the new \
+                        Apple TV screensaver that displays the Aerial \
+                        movies Apple shot over New York, San \
+                        Francisco, Hawaii, China, etc.
+
+# uses git submodules, not included in archive
+fetch.type              git
+
+post-fetch {
+    # fetch the only required submodule; cloning Sparkle fails
+    system -W ${worksrcpath} "git submodule update --depth 1 --init Extern/OAuthSwift"
+}
+
+# add missing internal dependency to simplify build
+# https://github.com/JohnCoates/Aerial/pull/1085
+patchfiles             patch-xcode-dependency.diff
+
+# although the binary works on 10.9 and later, it doesn't build on
+# anything earlier than 10.15
+pre-fetch {
+    if {${os.subplatform} eq "macosx" && [vercmp ${macosx_version} 10.15] < 0} {
```

Comment:
Please also set `known_fail yes` when this condition is true, so that the buildbot will know to skip the build. Move the condition to the top level and move the pre-fetch block inside it.